src/Settings/Builders/Make.hs

```diff
@@ -10,16 +9,9 @@ makeBuilderArgs = do
     threads    <- shakeThreads <$> expr getShakeOptions
     gmpPath    <- expr gmpBuildPath
     libffiPath <- expr libffiBuildPath
-    ghcPath    <- expr $
-      (-/-) <$> topDirectory <*> builderPath (Ghc CompileHs Stage2)
-    perlPath   <- expr $ builderPath Perl
     let t = show $ max 4 (threads - 2) -- Don't use all Shake's threads
```

Comment:
@alpmestan By the way, do you think `nofib` could benefit from using parallelism? Perhaps, we could pass `t` to its Make invocations in the examples below?

Reply:
@snowleopard I thought about it but I don't think we want to do that. First, [the nofib trac page](https://ghc.haskell.org/trac/ghc/wiki/Building/RunningNoFib) does not mention any parallelism variable/knob. Second, nofib is after all a benchmark suite, so we quite likely don't want to run many of them simultaneously as this could have quite an impact on their respective performance and therefore would make the measurements "useless".

Reply:
@alpmestan Thanks, I see.